CHANGELOG.md

```diff
@@ -16,6 +16,7 @@ This project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.htm
 ### Changed
 
 - Jaeger exporter takes into additional 70 bytes overhead into consideration when sending UDP packets (#2489)
+- Jaeger exporter takes into additional 70 bytes overhead into consideration only when splitting spans into batches (#2512)
```

Comment:
```suggestion
- Jaeger exporter takes into additional 70 bytes overhead into consideration when sending UDP packets (#2489, #2512)
```

I think there's really only need for one entry here.

Reply:
good point 👍🏽